test/e2e/public-viewer/proxy_test.go

```diff
@@ -0,0 +1,181 @@
+package community_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/require"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	waitpoll "k8s.io/apimachinery/pkg/util/wait"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/api/v1alpha1"
+	commonauth "github.com/codeready-toolchain/toolchain-common/pkg/test/auth"
+	. "github.com/codeready-toolchain/toolchain-e2e/testsupport"
+	authsupport "github.com/codeready-toolchain/toolchain-e2e/testsupport/auth"
+	"github.com/codeready-toolchain/toolchain-e2e/testsupport/spacebinding"
+	"github.com/codeready-toolchain/toolchain-e2e/testsupport/tiers"
+	"github.com/codeready-toolchain/toolchain-e2e/testsupport/wait"
+)
+
+type proxyUser struct {
+	expectedMemberCluster *wait.MemberAwaitility
+	username              string
+	token                 string
+	identityID            uuid.UUID
+	signup                *toolchainv1alpha1.UserSignup
+	compliantUsername     string
+}
+
+// tests access to community-shared spaces
+func TestPublicViewerProxy(t *testing.T) {
+	// given
+
+	// make sure everything is ready before running the actual tests
+	awaitilities := WaitForDeployments(t)
+	hostAwait := awaitilities.Host()
+	memberAwait := awaitilities.Member1()
+	// we create a space to share , a new MUR and a SpaceBindingRequest
+	space, _, _ := NewSpaceBindingRequest(t, awaitilities, memberAwait, hostAwait, "admin")
+
+	communityUser := &proxyUser{
+		expectedMemberCluster: memberAwait,
+		username:              "community-user",
+		identityID:            uuid.New(),
+	}
+	createAppStudioUser(t, awaitilities, communityUser)
+
+	communityUserProxyClient, err := hostAwait.CreateAPIProxyClient(t, communityUser.token, hostAwait.APIProxyURL)
+	require.NoError(t, err)
+
+	t.Run("space is flagged as community", func(t *testing.T) {
+		// when
+		_ = CreateCommunitySpaceBinding(t, hostAwait, space.Name, space.Namespace)
```

Comment:
```suggestion
                err = CreateCommunitySpaceBinding(t, hostAwait, space.Name, space.Namespace)
```

Reply:
thanks, Rafaela! I fixed these lines in 86d96ccf3eff37007feb199ae383090c7ca3f338